apps/web/src/components/ClickableLogo/ClickableLogo.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import ClickableLogo from './ClickableLogo';
+
+export default {
+  title: 'Components/ClickableLogo',
+  component: ClickableLogo,
+};
+
+interface Props {
+  onClick: () => void;
+}
+
+const Template = ({ onClick }: Props) => <ClickableLogo onClick={onClick} />;
```

Comment:
Remember, this is not the default way of defining a story. Unless we need to pass in a child component stories should always be defined using the `bind({})` syntax. 

And if you do it that way you don't need to define a special Props type. 

Also, if you ever need to use the `Props` type from a component outside of the component you should export the type from the component file and import it where you need it. If you do it the way you've done it here, you're creating an unnecessary dependency. What happens if we add a new `Prop` to `ClickableLogo`? You'd have to update it in `ClickabaleLogo.tsx` and here in the story. That could get messy.